applications/mne_rt_server/connectors/Neuromag/dacqserver.h

```diff
@@ -42,49 +41,44 @@
 // INCLUDES
 //=============================================================================================================
 
-#include "types_definitions.h"
-#include <fiff/fiff_info.h>
-#include <fiff/fiff_tag.h>
-
 
 //*************************************************************************************************************
 //=============================================================================================================
 // QT INCLUDES
 //=============================================================================================================
 
 #include <QThread>
-
-#include <QTcpSocket>
-
-#include <QByteArray>
+#include <QSharedPointer>
+#include <QPointer>
 
 
 //*************************************************************************************************************
 //=============================================================================================================
-// DEFINE NAMESPACE NeuromagPlugin
+// FORWARD DECLARATIONS
 //=============================================================================================================
 
-namespace NeuromagPlugin
-{
+namespace FIFFLIB {
+    class FiffInfo;
+}
 
 
 //*************************************************************************************************************
 //=============================================================================================================
-// USED NAMESPACES
+// DEFINE NAMESPACE NEUROMAGRTSERVERPLUGIN
 //=============================================================================================================
 
-using namespace FIFFLIB;
+namespace NEUROMAGRTSERVERPLUGIN
+{
 
 
 //*************************************************************************************************************
 //=============================================================================================================
-// FORWARD DECLARATIONS
+// NEUROMAGRTSERVERPLUGIN FORWARD DECLARATIONS
```

Comment:
Why not just keeping FORWARD DECLARATIONS

Reply:
These are the forward declarations inside of the NEUROMAGRTSERVERPLUGIN namespace. The global forward declarations above are specified as FORWARD DECLARATIONS